docs/source/cli.rst

```diff
@@ -0,0 +1,118 @@
+
+.. _cli:
+
+Command Line Interface
+=======================================
+
+.. highlight:: console
+
+When git-machete is installed it is added to your path, so it can be called from command line: `git-machete some_command some_command_options`. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
+
+git-machete commands:
```

Comment:
Nuance: `format` and `hooks` aren't commands, just help topics :goncern:

Reply:
Would you like me to change this line?
I can split the list into `commands` and` help topics` or rename it to `git machete commands and help topics`.

Reply:
Yeah, let's go for the latter, rename